cmd/xslttransformation-adapter/Dockerfile

```diff
@@ -14,24 +15,33 @@ RUN go build -o /xslttransformation-adapter ./cmd/xslttransformation-adapter
 # (i) Entries marked with a '*' are not included in the 'distroless:base' image.
 #     linux-vdso.so.1
 #   * libxml2.so.2 => /usr/lib/x86_64-linux-gnu/libxml2.so.2
+#   * libxslt.so.1 => /usr/lib/x86_64-linux-gnu/libxslt.so.1
+#   * libexslt.so.0 => /usr/lib/x86_64-linux-gnu/libexslt.so.0
 #     libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0
 #     libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6
 #     libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2
 #   * libicuuc.so.67 => /usr/lib/x86_64-linux-gnu/libicuuc.so.67
 #   * libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1
 #   * liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5
 #     libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6
+#   * libgcrypt.so.20 => /usr/lib/x86_64-linux-gnu/libgcrypt.so.20
 #     /lib64/ld-linux-x86-64.so.2
 #   * libicudata.so.67 => /usr/lib/x86_64-linux-gnu/libicudata.so.67
 #   * libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6
 #   * libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1
+#   * libgpg-error.so.0 => /lib/x86_64-linux-gnu/libgpg-error.so.0
+
 
 FROM gcr.io/distroless/base-debian11:nonroot
 
 # (!) COPY follows symlinks
 COPY --from=builder \
     /usr/lib/x86_64-linux-gnu/libxml2.so.2 \
+    /usr/lib/x86_64-linux-gnu/libxslt.so.1 \
+    /usr/lib/x86_64-linux-gnu/libexslt.so.0 \
     /usr/lib/x86_64-linux-gnu/libicuuc.so.67 \
+    /usr/lib/x86_64-linux-gnu/libgcrypt.so.20 \
+    /lib/x86_64-linux-gnu/libgpg-error.so.0 \
```

Comment:
I really hope you are copying all dependent libraries as well otherwise I can see some very interesting runtime behavior due to unresolved symbols.

Reply:
For my own knowledge, does the output of `ldd` include all libraries the compiled app depends on?

Reply:
It does as well as whether the library is resolvable.